yr/tutorials/AlexaHelloWorld.md

```diff
@@ -168,14 +175,14 @@ Your current screen should look like the following:
 
 ## Testing your Skill
 
-Now it’s time to test our Alexa skill! If you have an Alexa-enabled device handy and [set up](https://docs.google.com/document/d/15SbjSasmgiCibz_NK8rS0Le1hZ1dZNZSb7a5JIT8H1c/edit), say *“Alexa, open hello world greeter”,* and then *“Alexa, tell hello world greeter to say hello.”* to it and hear the response, which should be “hello world”, as we programmed it.
+Now it’s time to test our Alexa skill! If you have an Alexa-enabled device handy and [set up](../images/alexa_hello_world/setup.pdf), say *“Alexa, open hello world greeter”,* and then *“Alexa, tell hello world greeter to say hello.”* to it and hear the response, which should be “hello world”, as we programmed it.
 
 If you don’t have an Alexa-enabled device on hand, App Inventor allows you to simulate an Alexa using your custom Alexa skill right in your browser! Simply type in the textbox at the bottom, *“Alexa, ask hello world greeter to say hi.”* The response should be what we plugged into the <strong>say</strong> block earlier, *“hello world.”*
 
 ![Sayhello](../images/alexa_hello_world/sayhello.png){:.enlargeImage} 
 <p style="padding-bottom: 7px"></p>
 
-Note: If you are interested in testing in other ways, feel free to explore this [document](https://docs.google.com/document/u/0/d/1ISS1lI1MK7PCehxCRS3eKQkBiA0fjeXwuL_WlAY-fQU/edit)!
+Note: If you are interested in testing in other ways, feel free to explore this [document](../images/alexa_hello_world/testing.pdf)!
```

Comment:
In the linked pdf document please remove the links to:
a. Hello World
b. My Calculator

as they are pointing to Vik's older documents etc.  I could not locate the original doc may need to ask Vik.

Reply:
Check if there are other places we have this issue.  None of the links in any doc we share should lead Vik's old docs.